internal/gamepad/gamepad.go

```diff
@@ -187,11 +187,76 @@ type Gamepad struct {
 	native nativeGamepad
 }
 
+type mappingInput interface {
+	Pressed() bool
+	Value() float64 // Normalized to range: 0..1.
+}
+
+type axisMappingInput struct {
+	g    nativeGamepad
+	axis int
+}
+
+func (a axisMappingInput) Pressed() bool {
+	const axisToButtonThreshold = 30 / 255.0
```

Comment:
Let's integrate `threshold` in `gamepad_xbox_windows.go`

Reply:
Done.